### ISSUE TITLE
Selenide hover doesn't work in Safari

### ISSUE THREAD

**Issue by MaryiaBV** [selenide/selenide#1890]
## The problem
Hello all! I am faced with issue for hovering feature on Safari browser. Seems that it doesn't work.

## Details
I need to hover over an element _(span or div)_ to see a button near that element. But I receive the MoveTargetOutOfBoundsException.
I tried the next options: selenideElement.hover(), Actions moveToElement, offSets. But those methods throw the same error and don't work.
My element is in center of the page. It is visible. I use scrollIntoView before hovering.

Also I noticed that if an element is near the end of page view and I do the scroll before hovering, MoveTargetOutOfBoundsException will happen. But if the element has been already in the center of the page (scroll is not made), exception will not be thrown but hovering also will not happen.

## Code To Reproduce Issue
```
Selenide.executeJavaScript("arguments[0].scrollIntoView({behavior: 'auto', block: 'center', inline: 'center'});", hoverEl);
hoverEl.hover().$("button.add-card").click();
```

Also, I tried the next code:
```
Actions actions = new Actions(WebDriverRunner.getWebDriver());
actions.moveToElement(hoverEl).perform();
hoverEl.$("button.add-card").click();
```

## Tell us about your environment
* **Selenide Version**: 6.6.1
* **Safari Version**: 15.5
* **Selenium WebDriver Version**: 4.3.0
* **OS Version**: MacOS Monterey 12.4

**Comment by MaryiaBV** [comment]
Hello Selenide Team! Have you had the chance to look at that issue?

**Comment by asolntsev** [comment]
@MaryiaBV no. I personally didn't even try. Many things don't work normally in safari driver, I lost the hope a long time ago. :(
